.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "0.0.1"
+        description: "Version of the release"
+        required: true
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ inputs.version }}
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: configure git
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
```

Comment:
Are these steps actually necessary? IIRC there are some defaults set up that you can use, no?

Also, whose email is this? Yours?

Reply:
I'll do some testing without this step! I'm guessing the `user.*` of whoever runs the workflow will be used, but am not certain...

This is [a somewhat secret email](https://github.com/orgs/community/discussions/26560#discussioncomment-3252339) used for @github-actions[bot] based on the ID and username - I tend to use it for workflow related things like this, but no problem removing it if all still works!